Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:17.04
```

Comment:
With `Python:3.6` as base image you saving the rest .

Reply:
Good one. Will do it.